src/__tests__/validators.test.ts

```diff
@@ -241,6 +242,39 @@ describe('validateMap', () => {
   });
 });
 
+describe('validateFile', () => {
+  test('should handle json files', () => {
+    expect(validateFile('file(json)', 'testFile.json')).toEqual(true);
+  });
+
+  test('should reject unknown file types', () => {
+    expect(validateFile('file(unknown)', 'testFile')).toEqual(
+      'Invalid file format value. Received "unknown"'
+    );
+  });
+
+  test('should reject absolute file paths', () => {
```

Comment:
this should fail, but not due to absolute path. It should be looking to make sure the filename includes the extension for the inputted file(*) type, in our case currently `.json`